packages/cli/src/commands/build.ts

```diff
@@ -281,24 +281,12 @@ export default async function main(client: Client) {
       cwd: cwd,
     });
   } else if (fs.existsSync(join(cwd, 'package.json'))) {
-    result = await runPackageJsonScript(
+    await runPackageJsonScript(
       client,
       cwd,
       ['vercel-build', 'now-build', 'build'],
       spawnOpts
     );
-  } else {
-    // no package.json exists and no build command present
-    result = true;
-  }
-
-  if (!result) {
```

Comment:
We removed this error message because it's possible you have a `package.json` and no build script defined (i.e. I want to have deps in my middleware or API file).